tsl/test/expected/dist_ddl.out

```diff
@@ -34,9 +35,6 @@ CALL distributed_exec('CREATE TABLE local(time timestamptz, measure int)', '{ "d
 CALL distributed_exec($$ SELECT create_hypertable('local', 'time') $$, '{ "data_node_1", "data_node_3" }');
 -- Import testsupport.sql file to data nodes
 \unset ECHO
--- This SCHEMA will not be created on data nodes
-CREATE SCHEMA disttable_schema AUTHORIZATION :ROLE_1;
-CREATE SCHEMA some_schema AUTHORIZATION :ROLE_1;
```

Comment:
`some_schema` is  now created above instead, but without AUTHORIZATION. Is this intentional?